### PR TITLE
FE-2: Initial json data package with example checking of question type in walkthroughes

### PR DESCRIPTION
**Pull request by nicholaslyon-dt**
**Purpose**
1. Setup initial data access paradigm for the web app and new data package
2. Import and make Types for walkthrough 1 - 9.9.9 intro questions 0.1-0.8 and results 0.0-0.5

**Notes**
Final report PDF with all Questions and Logic for both walkthroughes can be found here in Google Drive:
https://drive.google.com/file/d/1Wv9hqNGLy-mo1B5z8dMD1Enf96ggQltX/view?usp=drive_link

Questions and Logic covered in this PR can be found on pages 16-18
Results covered in the PR can be found on page 22

I also stumped out a way to access Building Code data for future use.

**Testing/Feedback**
1. Walkthrough content accuracy vs PDF
2. Walkthrough logic accuracy vs PDF
3. Walkthrough JSON structure
4. Walkthrough JSON logic paradigms
5. Data package setup
6. Data package Type setup and Type checking

### CODE REVIEW

File: packages/data/src/useBuildingCodeData.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+interface BuildingCodeJSONType {
+  title: string;
+  content: string;
+}
+
+const BuildingCodeJSONData: Record<BuildingCodes, BuildingCodeJSONType> = {
+  "9.9.9": {
+    title: "9.9.9",
+    content:
+      "Except as provided in Sentences (2) and (3), every dwelling unit containing more than 1 storey shall have exits or egress doors located so that it shall not be necessary to travel up or down more than 1 storey to reach a level served by",
```
Comment (laurenrevere) [inline packages/data/src/useBuildingCodeData.ts:19]: Is this the full content? That doesn't seem like the end of the sentence.

**Review by laurenrevere — APPROVED** [review]
Haven't checked all of the content yet, but will as we build this out.

File: packages/data/src/useBuildingCodeData.ts (hunk 1 above)
Comment (nicholaslyon-dt) [reply packages/data/src/useBuildingCodeData.ts:19]: No not actual content at all, just example text for the interface. Content for the Building Code JSON hasn't been brought over yet.

Comment (laurenrevere) [reply packages/data/src/useBuildingCodeData.ts:19]: Okay cool.

**Pull request merged**